happi/cli.py

```diff
@@ -321,41 +321,41 @@ def edit(ctx, name: str, edits: List[str]):
                 raise click.Abort()
         else:
             try:
-                value = device._info_attrs[field].enforce_value(value)
+                value = item._info_attrs[field].enforce_value(value)
             except ValueError:
                 logger.error(f'Error enforcing type for value: {value}')
                 raise click.Abort()
 
         # set field
         try:
-            getattr(device, field)
+            getattr(item, field)
             logger.info(f'Setting {name}.{field}: {value}')
-            setattr(device, field, value)
+            setattr(item, field, value)
         except Exception as e:
             logger.error(f'Could not edit {name}.{field}: {e}')
             raise click.Abort()
 
-    device.show_info()
-    if click.confirm('Please confirm the device info is correct'):
-        logger.info('Editing device')
-        device.save()
+    item.show_info()
+    if click.confirm('Please confirm the item info is correct'):
+        logger.info('Editing item')
+        item.save()
     else:
         logger.info('Aborting')
 
 
 @happi_cli.command()
-@click.argument('device_names', nargs=-1)
+@click.argument('item_names', nargs=-1)
 @click.pass_context
-def load(ctx, device_names: List[str]):
+def load(ctx, item_names: List[str]):
     """Open IPython terminal with DEVICE_NAMES loaded"""
```

Comment:
```suggestion
    """Open IPython terminal with ITEM_NAMES loaded"""
```

Reply:
Agreed on the change - but in the broader context of parameter names in docstrings:

I've searched `click` documentation and I can't find a reference to how we're intended to format the docstrings here to refer to a parameter. Does it matter? Is it a convention? I had asked @ZLLentz to change `happi benchmark` to `--duration` instead of `DURATION` because (1) I thought it looked ugly and (2) it wasn't really useful to match up with the actual CLI argument.

Reply:
When I first wrote the docstrings I followed this very rough guidance https://click.palletsprojects.com/en/8.1.x/documentation/#documenting-arguments

Apparently there's some Unix convention that I never verified?

Reply:
Hmm, the linked documentation makes me think we should be adding `metavar` to these and using that reference in the docstring:

```python
@click.command(options_metavar='<options>')
@click.option('--count', default=1, help='number of greetings',
              metavar='<int>')
@click.argument('name', metavar='<name>')
def hello(count, name):
    """This script prints hello <name> <int> times."""
    for x in range(count):
        click.echo(f"Hello {name}!")
```

`happi profile` for `--profiler` shows:
```
  -p, --profiler TEXT  Select which profiler to use.
```

Reply:
At risk of making too big of a deal out of this in the _entirely wrong PR_ I'm going to just accept your suggestion and we can circle back some other time.

Distractions abound...

Reply:
I like that, maybe for another PR 👍